src/chaplin/views/collection_view.coffee

```diff
@@ -474,10 +474,11 @@ module.exports = class CollectionView extends View
     # Insert the view into the list.
     list = if $ then @$list else @list
```

Comment:
It seems that this variable should go into 'if included' block at the bottom, since it's used only there.